src/js/worklets/header_animations.js

```diff
@@ -0,0 +1,23 @@
+import {
+  useDerivedValue,
+  interpolate,
+  interpolateColor,
+  Extrapolation,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+
+export function blurAmount(sharedValue) {
```

Comment:
Suggested syntax:

```
export const blurAmount = (sharedValue) => 
  useDerivedValue(() => 
    parseInt(
      interpolate(
        sharedValue.value, 
        [CLAMP_MIN, CLAMP_MAX], 
        [BLUR_MIN, BLUR_MAX], 
        Extrapolation.CLAMP
      )
    )
  );
```

Reply:
I'd also suggest to keep the `use` prefix for hooks, otherwise they might be misused deeper in the functions by mistake.

Reply:
@clauxx That's a good suggestion, I'll appy it